src/common/retryer.js

```diff
@@ -45,8 +45,11 @@ const retryer = async (fetcher, variables, retries = 0) => {
     // prettier-ignore
     // also checking for bad credentials if any tokens gets invalidated
     const isBadCredential = err.response.data && err.response.data.message === "Bad credentials";
+    const isAccountSuspended =
+      err.response.data &&
+      err.response.data.message === "Sorry. Your account was suspended.";
 
-    if (isBadCredential) {
+    if (isBadCredential || isAccountSuspended) {
       logger.log(`PAT_${retries + 1} Failed`);
```

Comment:
Seems good, but again, would be better if we could keep different features/fixes to different PRs. Would be easier to merge in that case.

Reply:
@Zo-Bro-23 Sorry, that code was due to the PR not being on the main branch. I now rebased it.